packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -89,7 +89,7 @@ const AddOrManageAccounts = props => {
 
     if (is_loading) return <LoadingModal />;
 
-    const fiat_section = has_fiat && (
+    const fiat_section = (
```

Comment:
`has_fiat` condition was added to fix a different issue.
Adding a new currency when we select crypto as the first currency didn't work without `has_fiat` condition check

Reply:
We check for `has_fiat` in line 132 and 171, why do we need to double check it here?

Reply:
It fails when the jsx is created for `fiat_section`. 
`current_fiat_currency` will be undefined and it fails at this line. We can also add optional chaining to fix it
```
fiat_currency: props.current_fiat_currency.toUpperCase(),
```

Reply:
You are right, but we use `fiat_section` in 2 lines `132` & `171` and in both places we have a condition for `has_fiat`.
so it is redundant to check for it again in `fiat_section` definition

Reply:
`fiat_section` is inline jsx, it will be created every time. So, the conditions at lines `132` and `171` don't matter